main.go

```diff
@@ -130,6 +130,17 @@ func main() {
 		}()
 	}
 
+	// Fail fast if the AWS credentials are not present.
+	awsCredentialsPath := os.Getenv("AWS_SHARED_CREDENTIALS_FILE")
+	if len(awsCredentialsPath) == 0 {
+		awsCredentialsPath = "/home/.aws/credentials"
+	}
+	_, err := os.Stat(awsCredentialsPath)
+	if err != nil {
+		setupLog.Error(err, "AWS credentials not found")
```

Comment:
The file is not required - if you're using an IAM instance profile, for example. I'm not sure we can error 100% of the time if it's missing. @randomvariable any suggestions?

Reply:
Thanks for the quick feedback @ncdc! 🙂 Ah, I forgot about that - looking at the default manifests and docs, I assumed that the credential file was required. 

I do think an explicit check (in whatever form) would help the debuggability, but it might be a bit more complicated than I thought. 

My bad, feel free to close this PR!

Reply:
Yes, that's correct. I've started writing an FAQ explaining this behaviour.
Sorry about that!

/close